src/inspector/inspectorhandler.cpp

```diff
@@ -544,6 +544,25 @@ static QList<InspectorCommand> s_commands{
                        return obj;
                      }},
 
+    InspectorCommand{"is_feature_enabled", "Check if a feature is enabled", 1,
+                     [](InspectorHandler*, const QList<QByteArray>& arguments) {
+                       QJsonObject obj;
+                       QString featureName = arguments[1];
+                       auto featureModel = FeatureModel::instance();
+                       QObject* featureObj = featureModel->get(featureName);
+
+                       if (!featureObj) {
+                         obj["error"] = "Feature does not exist";
+                         return obj;
+                       }
+
+                       Feature* feature = static_cast<Feature*>(featureObj);
```

Comment:
Couldn't this be replaced with:
`const Feature* feature = Feature::getOrNull(featureName);
                       if (!feature) {`